meta/main.yml

```
@@ -1,68 +1,24 @@
-# SPDX-License-Identifier: MIT
 ---
 galaxy_info:
-  # Replace with role's author name:
-  author: John Doe <jdoe@corp.com>
-  # Replace with the real description of what is role's purpose:
-  description: Basic template for Linux system roles
-  # Replace with the company the role's author is member of:
-  company: John Doe, Inc.
+  author: Todd Gill <tgill@redhat.com>
+  description: System Snapshot Role
+  galaxy_tags:
+    - system
+    - time
+    - redhat
+    - rhel
+    - fedora
+    - centos
 
-  # If the issue tracker for your role is not on github, uncomment the next
-  # line and provide a value
-  # issue_tracker_url: http://example.com/issue/tracker
-
-  # Some suggested licenses:
-  # - BSD (default)
-  # - MIT
-  # - GPLv2
-  # - GPLv3
-  # - Apache
-  # - CC-BY
+  company: Red Hat, Inc.
   license: MIT
-
   min_ansible_version: "2.9"
-
-  # Optionally specify the branch Galaxy will use when accessing the GitHub
-  # repo for this role. During role install, if no tags are available, Galaxy
-  # will use this branch. During import Galaxy will access files on this
-  # branch. If Travis integration is configured, only notifications for this
-  # branch will be accepted. Otherwise, in all cases, the repo's default branch
-  # (usually main) will be used.
-  # github_branch:
-
-  #
-  # platforms is a list of platforms, and each platform has a name and a list
-  # of versions.
-  #
-  # platforms:
-  # - name: Fedora
-  #   versions:
-  #   - all
-  #   - "25"
-  # - name: SomePlatform
-  #   versions:
-  #   - all
-  #   - "1.0"
-  #   - "7"
-  #   - "99.99"
   platforms:
-    # Replace the below with your platform list:
     - name: Fedora
       versions:
         - all
     - name: EL
       versions:
         - all
```

```suggestion
        - "8"
        - "9"
```
Best to use explicit EL versions - also add `- "7"` if the role will support EL7